packages/validator/src/validator.ts

```diff
@@ -19,12 +19,24 @@ import {toHexString} from "@chainsafe/ssz";
 import {ValidatorEventEmitter} from "./services/emitter";
 import {ValidatorStore, Signer} from "./services/validatorStore";
 import {computeEpochAtSlot, getCurrentSlot} from "@chainsafe/lodestar-beacon-state-transition";
+import {SecretKey} from "@chainsafe/bls";
+import {BlockDutiesService} from "./services/blockDuties";
+import {AttestationDutiesService} from "./services/attestationDuties";
+import {SyncCommitteeDutiesService} from "./services/syncCommitteeDuties";
+import {PubkeyHex} from "./types";
+
+export type SecretKeyInfo = {
+  secretKey: SecretKey;
+  keystorePath?: string;
+  unlockSecretKeys?: () => void;
+};
 
 export type ValidatorOptions = {
   slashingProtection: ISlashingProtection;
   dbOps: IDatabaseApiOptions;
   api: Api | string;
   signers: Signer[];
+  secretKeysInfo?: SecretKeyInfo[];
```

Comment:
I don't think this is used here? remove if possible, so the key manager becomes the sole 'owner' of this.

Reply:
Already removed. This diff is outdated.